scripts/quests/crystalWar/A_Little_Knowledge.lua

```diff
@@ -0,0 +1,143 @@
+-----------------------------------
+-- A Little Knowledge
+-----------------------------------
+-- !addquest 7 6
+-- Erlene      : !pos 376.936 -39.999 17.914 175
+-----------------------------------
+require('scripts/globals/npc_util')
+require('scripts/globals/quests')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.CRYSTAL_WAR, xi.quest.id.crystalWar.A_LITTLE_KNOWLEDGE)
+
+quest.reward =
+{
+    ki = xi.ki.GRIMOIRE,
+    title = xi.title.SCHULTZ_SCHOLAR,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and player:getMainLvl() >= xi.settings.main.ADVANCED_JOB_LEVEL
+        end,
+
+        [xi.zone.THE_ELDIEME_NECROPOLIS_S] =
+        {
+            ['Erlene'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(10, 1)
+                end
+            },
+
+            onEventFinish =
+            {
+                [10] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            }
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED
+        end,
+
+        [xi.zone.THE_ELDIEME_NECROPOLIS_S] =
+        {
+            ['Erlene'] =
+            {
+                onTrade = function(player, npc, trade)
+                    local mJob = player:getMainJob()
+                    if npcUtil.tradeHasExactly(trade, { { xi.items.VELLUM, 12 } }) then
+                        player:tradeComplete()
```

Comment:
npcUtil uses confirmItem, better practice to use confirmTrade (and would prefer at the event finish of event 12)